2018-October-30/writingSamples/heslin.tex

```diff
@@ -0,0 +1,170 @@
+been recently implicated in pre-frontal dependent cognitive tasks
+\textsuperscript{17}. However, precisely how D1DR-expressing cells of
+the LCN contribute to cognitive function and modulation of the frontal
+cortex is unclear.
+
+To address this, we will evaluate the role of LCN D1DR-expressing
+neurons in performance of an interval timing task. In interval timing
+tasks, subjects are presented with a cue that signals them to respond
+after a certain duration elapses (e.g., press a lever after 12 seconds).
+Interval timing performance is impaired in schizophrenia
+patients\textsuperscript{18--21}, depends on multiple nodes in our
+circuit of interest\textsuperscript{22,23}, and is impacted by
+manipulations of dopaminergic signaling\textsuperscript{24--26}.
+Successful interval timing performance requires recruitment and
+coordination of several different executive functions such as working
+memory, attention, and decision making. Importantly, interval timing
+task measures can be readily compared across animal models,
+neuropsychiatric patients, and healthy human comparisons. Therefore, it
+is an ideal type of task with which to probe our circuit and neuronal
+population of interest in rodent models. Specifically, we will
+investigate how LCN D1DR-expressing neurons contribute to performance in
+a peak interval timing task, and how performance correlates with
+population activity in the LCN and prelimbic cortex (PL), the rodent
+medial frontal cortex. Based on our prior work and the primary
+literature, our \textbf{\emph{overall hypothesis}} is that inhibitory
+D1DR-expressing cells of the LCN are necessary for successful interval
+timing through modulation of activity in the PL.
+
+\textbf{\emph{Aim 1. Determine if LCN D1DR-expressing neurons are
+necessary for interval timing}}
+
+The cerebellum and LCN specifically have been implicated in both
+sub-second and supra-second interval timing\textsuperscript{23,27--30}.
+Dopaminergic signaling is crucial for timing behavior and is one of
+several abnormal neurotransmitter systems in
+schizophrenia\textsuperscript{4,21}. The role of inhibitory
+D1DR-expressing neurons in the LCN is presently unknown. We will
+determine if LCN D1DR-expressing neurons are necessary for interval
+timing performance using optogenetically driven inactivation during the
+task. Additionally, synchronization of cortical pyramidal cell activity
+by inhibitory interneurons is associated with cognitive
+processing\textsuperscript{31,32}. Therefore, we will record local field
+potentials (LFPs) and single neurons during optogenetic inactivation to
+assess whether LCN D1DR-expressing neurons contribute to synchronized
+activity in the LCN and PL. Combining optogenetic and
+electrophysiological methods will allow us to test the \emph{hypothesis}
+that inactivation of LCN D1DR-expressing neurons results in interval
+timing deficits and decreased synchronized neuronal activity in the LCN
+and PL.
```

Comment:
perhaps tie this back to how it impacts schizophrenia or some way of saying why this is impactful.